src/components/Link/Link.js

```diff
@@ -48,6 +73,8 @@ const Link = ({
   forceOpenInSameTab = false,
   ...rest
 }) => {
+  const location = useLocation();
```

Comment:
Now that this component uses a hook which relies on looking up the location on the window object, we need to provide a window like environment for testing. Hence the switch to jest and JSDOM for the tests of this component. They are probably better tests now anyway, since they run in an environment more like userland.